lib/types/application/ApplicationBuilder.js

```diff
@@ -1,23 +1,5 @@
 const AbstractBuilder = require("../AbstractBuilder");
-
-const tasks = { // can't require index.js due to circular dependency
-	generateComponentPreload: require("../../tasks/bundlers/generateComponentPreload"),
-	generateFlexChangesBundle: require("../../tasks/bundlers/generateFlexChangesBundle"),
-	generateLibraryPreload: require("../../tasks/bundlers/generateLibraryPreload"),
-	generateManifestBundle: require("../../tasks/bundlers/generateManifestBundle"),
-	generateStandaloneAppBundle: require("../../tasks/bundlers/generateStandaloneAppBundle"),
-	generateBundle: require("../../tasks/bundlers/generateBundle"),
-	generateCachebusterInfo: require("../../tasks/generateCachebusterInfo"),
-	escapeNonAsciiCharacters: require("../../tasks/escapeNonAsciiCharacters"),
-	buildThemes: require("../../tasks/buildThemes"),
-	createDebugFiles: require("../../tasks/createDebugFiles"),
-	generateVersionInfo: require("../../tasks/generateVersionInfo"),
-	replaceCopyright: require("../../tasks/replaceCopyright"),
-	replaceVersion: require("../../tasks/replaceVersion"),
-	transformBootstrapHtml: require("../../tasks/transformBootstrapHtml"),
-	uglify: require("../../tasks/uglify"),
-	generateApiIndex: require("../../tasks/jsdoc/generateApiIndex")
-};
+const taskRepository = require("../../tasks/taskRepository");
```

Comment:
What about this? We only need that method. This would shorten the many occurrences of taskRepository.getTask.
```suggestion
const {getTask} = require("../../tasks/taskRepository");
```

Reply:
Done